Functions.js

```diff
@@ -99,6 +101,17 @@ function htmlDecode(str) {
     return s;
 }
 
+/**
+ * @function isVerifiedChat
+ * @description 判断一个 chat 是否是授权的 chat
+ * @param {Num} chatId - 需要判断的 chat id
+ * @returns {Boolean} - 是否是授权 chat
+ */
+function isVerifiedChat(chatId)
+{
+    return chatId in verifiedChatIdList;
```

Comment:
```suggestion
    return verifiedChatIdList.includes(chatId);
```

- `0 in ["a", "b"]` => `true`
- `1 in ["a", "b"]` => `true`
- `2 in ["a", "b"]` => `false`
- `"a" in ["a", "b"]` => `false`